### PR TITLE
Add a production-ready helm chart for deploying buildfarm on k8s

### PR DESCRIPTION
**Pull request by lshmouse**
It's very convenient to deploy an app on k8s using helm chart. 

So we add  a production-ready helm chart for bazel buildfarm and tested in our k8s cluster.

Suggestions are welcomed, thx~

### CODE REVIEW

**Comment by lshmouse** [comment]
@werkt @luxe Could you help to review this pull request? Thanks~

File: kubernetes/helm-charts/buildfarm/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+data:
+  config.yml: |-
+    {{- range $key, $value := .Values.config }}
+    {{- if kindIs "map" $value }}
+    {{- else }}
+    {{ $key }}: {{ $value }}{{- end }}
+    {{- end }}
+    backplane:
+      {{- if .Values.redis.enabled }}
+      redisUri: "redis://{{ printf "%s-redis-master.%s" (include "redis.fullname" .) (.Release.Namespace) }}:{{ "6379" }}"
```
Comment (werkt) [inline kubernetes/helm-charts/buildfarm/templates/configmap.yaml:14]: Others have indicated that there may need to be a scheme control here (for rediss://), and the port and presumably operator convention are hard coded.

**Review by werkt — APPROVED** [review]
Seems pretty well contained, I'll have to check these out running. Will wait to see if you can tweak the redis uri to be more parameterized, or if its expected that folks will just change this for deployments.

File: kubernetes/helm-charts/buildfarm/templates/configmap.yaml (hunk 1 above)
Comment (lshmouse) [reply kubernetes/helm-charts/buildfarm/templates/configmap.yaml:14]: ok. I will update the PR later

Comment (lshmouse) [reply kubernetes/helm-charts/buildfarm/templates/configmap.yaml]: @werkt The PR has been updated. Please help to review it, thanks~

**Pull request merged**